src/App.vue

```diff
@@ -9,128 +9,135 @@
         hide-overlay
         app
       >
-
```

Comment:
When adding a single element, I expect to not change the whole component. Please update this file to reflect only the change to add the projects in the sidebar